src/components/__test__/Missions.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import { render } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import Missions from '../missions/Missions';
+import store from '../../redux/store';
+
+describe('Testing mission component', () => {});
+it('should render correctly Mission component', () => {
+  const tree = render(
+    <Provider store={store}>
+      <Missions />
+    </Provider>,
+
+  );
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] Great job so far 👏🏻 However you do have some failure linter tests. 👇🏻  Kindly fix this 💯 👍🏻 

![Screenshot from 2023-01-05 22-17-34](https://user-images.githubusercontent.com/79058364/210872181-03d56e32-fe34-413f-b6cf-d6c454b5d520.png)

Reply:
@BenMukebo I have now fixed all the tests. They should be passing and running alright.